src/test/java/api/support/fixtures/RequestQueueFixture.java

```diff
@@ -0,0 +1,50 @@
+package api.support.fixtures;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.TimeUnit;
+
+import org.folio.circulation.support.http.client.OkapiHttpClient;
+import org.folio.circulation.support.http.client.Response;
+import org.folio.circulation.support.http.client.ResponseHandler;
+
+import api.support.http.InterfaceUrls;
+import io.vertx.core.json.JsonObject;
+
+public class RequestQueueFixture {
+  private final OkapiHttpClient okapiHttpClient;
+
+  public RequestQueueFixture(OkapiHttpClient client) {
+    this.okapiHttpClient = client;
+  }
+
+  public Response attemptReorderQueue(String itemId, JsonObject reorderQueue) throws Exception {
```

Comment:
It throws `InterruptedException, ExecutionException, TimeoutException`

Reply:
I think it is clearer for the client to specify the specific checked exceptions that can occur, rather than a general exception, which provides very little information